tests/utils/test_uri.py

```diff
@@ -23,14 +24,19 @@ def test_extract_db_type_from_uri():
             extract_db_type_from_uri(unsupported_db)
 
 
-def test_get_db_profile_from_uri_casing():
-    assert get_db_profile_from_uri('databricks://aAbB') == 'aAbB'
-
-
-def test_get_db_path_info_from_profile():
-    assert get_db_path_info_from_uri('databricks://profile/prefix') == 'prefix'
-    assert get_db_path_info_from_uri('nondatabricks://profile/prefix') is None
-    assert get_db_path_info_from_uri('databricks://profile') is None
+def test_get_db_info_from_uri():
```

Comment:
could just parameterize and check both the profile and path values for each one?